src/workers/parser/tests/__snapshots__/closest.spec.js.snap

```diff
@@ -93,12 +92,9 @@ FunctionDeclaration (9:0,11:1)
   body: BlockStatement (9:25,11:1)
     body:
       - ReturnStatement (10:2,10:15)
-          argument: StringLiteral (10:9,10:14)
-            extra:
-              raw: '"meh"'
-              rawValue: 'meh'
+          argument: Literal (10:9,10:14)
```

Comment:
Not sure why the changes here happened bu they seem like they don't matter too much.